### PR TITLE
jp6 - dual camera configuration

### PR DESCRIPTION
**Pull request by Arun-Prasad-V**
Continuation of https://github.com/IntelRealSense/realsense_mipi_platform_driver/pull/223.

### CODE REVIEW

File: apply_patches_ext.sh [hunk 1]
```diff
@@ -60,7 +60,7 @@ fi
 cp $DEVDIR/kernel/realsense/d4xx.c $DEVDIR/$1/${D4XX_SRC_DST}/drivers/media/i2c/
 if [[ "$JETPACK_VERSION" == "6.0" ]]; then
     # jp6 overlay
-    cp $DEVDIR/hardware/realsense/tegra234-camera-d4xx-overlay.dts $DEVDIR/$1/hardware/nvidia/t23x/nv-public/overlay/
+    cp $DEVDIR/hardware/realsense/tegra234-camera-d4xx-overlay*.dts $DEVDIR/$1/hardware/nvidia/t23x/nv-public/overlay/
```
Comment (Nir-Az) [inline apply_patches_ext.sh:63]: Can you also check if "apply_patches.sh" need it?

Comment (Arun-Prasad-V) [reply apply_patches_ext.sh:63]: "apply_patches.sh" file is already handled in https://github.com/IntelRealSense/realsense_mipi_platform_driver/pull/223.

**Review by Nir-Az — APPROVED** [review]

**Comment by Nir-Az** [comment]
Maybe we need to align Jenkins API calls with GHA?

**Comment by Arun-Prasad-V** [comment]
> Maybe we need to align Jenkins API calls with GHA?

Actually, one follows manual source sync steps and the other uses `setup_workspace.sh` script. It is good to test both scenarios. We can either update the GHA to test both or revamp the `apply_patches.sh` and `apply_patched_ext.sh` scripts such that the common steps present at one place.

**Comment by Nir-Az** [comment]
> > Maybe we need to align Jenkins API calls with GHA?
> 
> Actually, one follows manual source sync steps and the other uses `setup_workspace.sh` script. It is good to test both scenarios. We can either update the GHA to test both or revamp the `apply_patches.sh` and `apply_patched_ext.sh` scripts such that the common steps present at one place.

Got it, thanks

**Pull request merged**